runtime/syntax/arduino.yaml

```diff
@@ -4,7 +4,7 @@ detect:
     filename: "\\.?ino$"
 
 rules:
-    - identifier: "\\b[A-Z_][0-9A-Z_]+\\b" 
+    - identifier: "\\b[A-Z_][0-9A-Z_]+\\b"
 
       ## 
```

Comment:
grep has told me about a remaining trailing space at this line.

Reply:
I've just tried and it told me about trailing space after `## ` which it true... But it is one of the changes of commit. Are you sure you grepped on PR's branch?

Reply:
> But the reported by you space is one of the changes of commit. Are you sure you grepped on PR's branch?

This is most probably caused by the scope of Github, but @dmaluka was most probably referring to line #9.

Reply:
Yep, I was referring to line 9.